vcd/resource_vcd_org.go

```diff
@@ -194,6 +195,12 @@ func resourceOrgCreate(ctx context.Context, d *schema.ResourceData, m interface{
 
 	if err != nil {
 		log.Printf("[DEBUG] Error creating Org: %s", err)
+		// Some 10.4 VCD versions have a bug that fail when creating a disabled Org
+		if !isEnabled && strings.Contains(err.Error(), "com.vmware.vcloud.common.model.oauth.oidc.OidcAuthorizationModel") {
+			err = fmt.Errorf("%s\n\nThis version of VCD has a bug which prevents creating a disabled Org\n"+
```

Comment:
I wonder if the error should come after the clarification? As it is a bit unintelligible Java stacktrace.
cc: @dataclouder @adezxc

Reply:
Final decision is to mask the error (which is too verbose for a regular user) and show our own:
```
│ Error: [org creation] error creating Org org1:
│ this version of VCD (10.4.2.21595985) has a bug that prevents creating a disabled Org
│ If you need to disable it, please create an enabled Org and then disable it using update
│
│   with vcd_org.org1,
│   on config.tf line 27, in resource "vcd_org" "org1":
│   27: resource "vcd_org" "org1" {
```